kube/resources/bootloader.yaml

```diff
@@ -1,40 +1,43 @@
-apiVersion: v1
-kind: Pod
+apiVersion: batch/v1
+kind: Job
 metadata:
   name: airbyte-bootloader
 spec:
-  restartPolicy: Never
-  containers:
-    - name: airbyte-bootloader-container
-      image: airbyte/bootloader
-      env:
-        - name: AIRBYTE_VERSION
-          valueFrom:
-            configMapKeyRef:
-              name: airbyte-env
-              key: AIRBYTE_VERSION
-        - name: DATABASE_HOST
-          valueFrom:
-            configMapKeyRef:
-              name: airbyte-env
-              key: DATABASE_HOST
-        - name: DATABASE_PORT
-          valueFrom:
-            configMapKeyRef:
-              name: airbyte-env
-              key: DATABASE_PORT
-        - name: DATABASE_PASSWORD
-          valueFrom:
-            secretKeyRef:
-              name: airbyte-secrets
-              key: DATABASE_PASSWORD
-        - name: DATABASE_URL
-          valueFrom:
-            configMapKeyRef:
-              name: airbyte-env
-              key: DATABASE_URL
-        - name: DATABASE_USER
-          valueFrom:
-            secretKeyRef:
-              name: airbyte-secrets
-              key: DATABASE_USER
+  ttlSecondsAfterFinished: 5
```

Comment:
As mentioned in the PR description, this is necessary to avoid an error being about the airbyte-bootloader job being immutable. This isn't an ideal solution, because this means that the bootloader pod is deleted after it completes, making its logs inaccessible through kube.

This was the only solution I could come up with that allowed us to still use `kubectl apply` without issue though, so it may be a worthwhile tradeoff. 

Definitely open to feedback here if there are any other options I haven't considered.

Reply:
can you add a comment explaining why we do this, please?

Reply:
Sure thing, done!